infra/terraform/modules/tezos_node/main.tf

```diff
@@ -33,6 +33,14 @@ resource "aws_launch_configuration" "tz_node" {
     mode              = var.TZ_MODE
   })
 
+  root_block_device {
+    volume_type           = "io1"
+    volume_size           = 10 #Gb
```

Comment:
il will be enough in the future ?

Reply:
for rolling node yes because we get only the last cycle